components/filemanager/events.md

```diff
@@ -328,7 +328,23 @@ The `OnModelInit` event fires when a new instance of the model is about to be cr
 
 ### OnDownload
 
-The `OnDownload` event fires before a file is to be downloaded, cancellable. Its event handler receives the updated `FileManagerDownloadEventArgs` as an argument. See the [example](#example).
+The `OnDownload` event fires before a file is to be downloaded. The event is cancellable. The event handler argument is an `FileManagerDownloadEventArgs` object. See the [example](#example).
```

Comment:
Can you try rephrasing "a file is to be downloaded"? Avoid the passive voice and try to describe the exact action that triggers the event.